plugins/poll/spec/controllers/posts_controller_spec.rb

```diff
@@ -228,54 +213,19 @@
             DiscoursePoll::Poll.vote(post_id, "poll", ["5c24fc1df56d764b550ceae1b9319125"], user)
           end
 
-          it "OP cannot change the options" do
+          it "cannot change the options" do
             put :update, params: {
               id: post_id, post: { raw: new_option }
             }, format: :json
 
-            expect(response).not_to be_success
+            expect(response).not_to be_successful
             json = ::JSON.parse(response.body)
             expect(json["errors"][0]).to eq(I18n.t(
-              "poll.edit_window_expired.op_cannot_edit_options",
+              "poll.edit_window_expired.cannot_edit_poll_with_votes",
               minutes: poll_edit_window_mins
             ))
           end
 
-          it "staff can change the options and votes are merged" do
-            log_in_user(Fabricate(:moderator))
-
-            put :update, params: {
-              id: post_id, post: { raw: new_option }
-            }, format: :json
-
-            expect(response.status).to eq(200)
-            json = ::JSON.parse(response.body)
-            expect(json["post"]["polls"]["poll"]["options"][1]["html"]).to eq("C")
-            expect(json["post"]["polls"]["poll"]["voters"]).to eq(1)
-            expect(json["post"]["polls"]["poll"]["options"][0]["votes"]).to eq(1)
-            expect(json["post"]["polls"]["poll"]["options"][1]["votes"]).to eq(0)
-          end
-
-          it "staff can change the options and anonymous votes are merged" do
-            post = Post.find_by(id: post_id)
-            default_poll = post.custom_fields["polls"]["poll"]
-            add_anonymous_votes(post, default_poll, 7, "5c24fc1df56d764b550ceae1b9319125" => 7)
-
-            log_in_user(Fabricate(:moderator))
-
-            put :update, params: {
-              id: post_id, post: { raw: new_option }
-            }, format: :json
-
-            expect(response.status).to eq(200)
-
-            json = ::JSON.parse(response.body)
-            expect(json["post"]["polls"]["poll"]["options"][1]["html"]).to eq("C")
-            expect(json["post"]["polls"]["poll"]["voters"]).to eq(8)
-            expect(json["post"]["polls"]["poll"]["options"][0]["votes"]).to eq(8)
-            expect(json["post"]["polls"]["poll"]["options"][1]["votes"]).to eq(0)
-          end
```

Comment:
Are these tests no longer required as well?

Reply:
Yeah, none are relevant anymore since we don't merge votes anymore (we delete them when the poll has changed)